pkg/api/routes_scene.go

```diff
@@ -42,13 +42,29 @@ func (rs sceneRoutes) Routes() chi.Router {
 // region Handlers
 
 func (rs sceneRoutes) Stream(w http.ResponseWriter, r *http.Request) {
+
+	//TODO get user agent and use to fine tune media compatibility
+	//ua := r.Header.Get("User-Agent")
+	//logger.Infof("[stream] user agent is: %s \n", ua)
+
 	scene := r.Context().Value(sceneKey).(*models.Scene)
 
+	// container isn't stored in db we need to get it every time
+	videoFile, err := ffmpeg.NewVideoFile(manager.GetInstance().FFProbePath, scene.Path)
+	if err != nil {
+		logger.Errorf("[stream] error reading video file: %s", err.Error())
+		return
+	}
+	container := ffmpeg.FfprobeToContainer[videoFile.Container]
+	if container == ffmpeg.Matroska {
+		container = ffmpeg.MagicContainer(scene.Path) // use magic file instead of ffprobe for matroska,webm
+	}
+
```

Comment:
Need to consider the performance impacts of this and compare against storing the container in the database instead.

Reply:
The container needs to be stored to the database eventually because of the generate transcodes task penalty .

If we want to merge this to the stable though as a bug fix release ( eg 0.1.2) we can leave the db implementation for 0.2.0 and a seperate PR. If we don't care about pushing to stable but only for dev / 0.2.0 then i should implement the db part in this PR.

The generate transcodes part has a noticable penalty and thus i  can revert it till the db implementation is ready.